### ISSUE TITLE
planning_timelimit is accessible only through FFTW class, should be also a function and in wrappers

### ISSUE THREAD

**Issue by smartass101** [hgomersall/pyFFTW#63]
The `planning_timelimit` is a useful parameter and at the moment it is accessible only through direct `pyfftw.FFTW` creation. A `set_planner_timelimit(max_seconds=None, precision=None)` function should be accessible (it would just map to the timelimit functions already written) and/or the argument should be available in other wrappers as outlined in #35.

Creating the function look pretty simple IMHO (as much of the code is already written) and it would not break backwards-compatibility with wrapper signitures, so I think it could be in the next release. Later on when #35 is resolved the wrappers could include it too as a part of the wrapper signiture change.

**Comment by hgomersall** [comment]
I don't understand the use of this... the planning time limit is only applicable the the creation of an `FFTW` object. Could you clarify when this is useful?

**Comment by hgomersall** [comment]
Having said that, I've just noticed the builders don't expose the time limit setting. I assume this is the use for it?

**Comment by smartass101** [comment]
Yes, that would be one of the obvious uses. It is applicable even to interfaces, because they use `'FFTW_MEASURE'` by default and it would help to be able to specify a limit. Actually, I think the interfaces module should either switch to `FFTW_ESTIMATE` by default, or keep the current default but set some time limit.

**Comment by hgomersall** [comment]
Yeah, good point. I think I agree as regards `FFTW_ESTIMATE` by default. Fancy submitting a PR for this stuff? (ideally with tests :)
